README.md

```diff
@@ -80,7 +80,7 @@ Hard-coding the Okta domain works for quick tests, but for real projects you sho
  
 ## Usage guide
 
-These examples will help you understand how to use this library. You can also browse the full [API reference documentation][dotnetdocs].
+These examples will help you understand how to use this library.
```

Comment:
Do we not have full reference docs at all anymore?

Reply:
We don't. As we don't have API docs for this SDK we agreed in removing this for now. You can see this in the thread above.